src/core/api/index.js

```diff
@@ -0,0 +1,20 @@
+import { Schema, arrayOf, normalize } from 'normalizr';
+
+// FIXME: Load this from something.
+const API_HOST = 'https://addons-dev.allizom.org';
```

Comment:
I filed https://github.com/mozilla/addons-frontend/issues/105 to support client-side configuration.